tests/network/secondary_network_dns/test_secondary_network_dns.py

```diff
@@ -56,12 +57,13 @@ def secondary_network_in_nslookup_output(
     # exists in the zones file):
     # wildcard.apps.{cluster_base_domain} is the API access from outside the cluster
     sampler = TimeoutSampler(
-        wait_timeout=TIMEOUT_40SEC,
-        sleep=TIMEOUT_5SEC,
+        wait_timeout=TIMEOUT_30SEC,
```

Comment:
But if the assumption is that the response can take 30 seconds to arrive, then limiting the wait to 30 seconds is a borderline, no?
Any reason why not continue waiting for 40 seconds before declaring failure?